copy/src/API.ts

```diff
@@ -0,0 +1,22 @@
+import { Question } from './redux/types';
+import { shuffleArray } from './utils'
+
+export enum Difficulty {
+    Easy = 'easy',
+    Medium = 'medium',
+    Hard = 'hard'
+};
+ 
+export const fetchQuizQuestions = async (amount: number, difficulty: Difficulty) => {
+    const endpoint = `https://opentdb.com/api.php?amount=${amount}&difficulty=${difficulty}&type=multiple`;
+    const data = await (await fetch(endpoint)).json();
+    return data.results.map((question: Question) => (
+        {
```

Comment:
you can add type to this new obj too, cause now it's untyped.
Nice way of doing it is adding it as return type of function, eg
async (..params): Promise<YourType> {....}